### PR TITLE
Fix PEP8 in tests

### PR DESCRIPTION
**Pull request by AlexandreAbraham**
Following Elvis remark: massive PEP8 correction.

### CODE REVIEW

**Comment by lesteve** [comment]
Out of interest did you run autopep8 or something else?

File: nilearn/datasets/tests/test_func.py [hunk 1]
```diff
@@ -129,7 +129,8 @@ def test_fetch_haxby():
     for i in range(1, 6):
         haxby = func.fetch_haxby(data_dir=tmpdir, n_subjects=i,
                                  verbose=0)
-        assert_equal(len(mock_url_request.urls), 1 + (i == 1))  # subject_data + md5
+        assert_equal(len(mock_url_request.urls), 1 + (i == 1))
+                     # subject_data + md5
```
Comment (lesteve) [inline nilearn/datasets/tests/test_func.py]: I guess this comment should be before the assert now that it is not inline anymore

File: nilearn/decoding/tests/test_same_api.py [hunk 2]
```diff
@@ -134,10 +134,15 @@ def test_graph_net_and_tvl1_same_for_pure_l1(max_iter=100, decimal=2):
 
 def test_graph_net_and_tvl1_same_for_pure_l1_logistic(max_iter=20,
                                                         decimal=2):
+<<<<<<< HEAD
     ###############################################################
     # graph_net_solver and tvl1_solver should give same results
+=======
+    #
+    # smoothlasso_solver and tvl1_solver should give same results
+>>>>>>> Super PEP8
```
Comment (lesteve) [inline nilearn/decoding/tests/test_same_api.py]: Oops ...

File: nilearn/decoding/tests/test_same_api.py [hunk 3]
```diff
@@ -134,10 +134,15 @@ def test_graph_net_and_tvl1_same_for_pure_l1(max_iter=100, decimal=2):
 
 def test_graph_net_and_tvl1_same_for_pure_l1_logistic(max_iter=20,
                                                         decimal=2):
+<<<<<<< HEAD
```
Comment (AlexandreAbraham) [inline nilearn/decoding/tests/test_same_api.py]: gloups :P

File: nilearn/plotting/tests/test_displays.py [hunk 4]
```diff
@@ -15,7 +15,7 @@
 from nilearn._utils.extmath import fast_abs_percentile
 
 
-###############################################################################
+#
```
Comment (lesteve) [inline nilearn/plotting/tests/test_displays.py]: Interesting "fix", is that autopep8 doing it? Arguably a better one would be to use `# ######` i.e. a space after the first hash.

Full disclosure, that may be just me but I am not a massive fan of these separating lines.

File: nilearn/tests/test_extmath.py [hunk 5]
```diff
@@ -12,7 +12,7 @@
 def test_fast_abs_percentile():
     data = np.arange(1, 100)
     for p in range(10, 100, 10):
-        yield nose.tools.assert_equal, fast_abs_percentile(data, p-1), p
+        yield nose.tools.assert_equal, fast_abs_percentile(data, p - 1), p
```
Comment (lesteve) [inline nilearn/tests/test_extmath.py]: so I discovered something the other day, PEP8 doesn't require you to have a space around binary operators. See [this](https://www.python.org/dev/peps/pep-0008/#other-recommendations) for example.

Interestingly, autopep8 fixes something that pep8 doesn't see as a violation ...

**Comment by AlexandreAbraham** [comment]
autopep8 FTW

**Comment by lesteve** [comment]
Alrighty merging ...

**Comment by lesteve** [comment]
Great stuff!

**Pull request merged**